feature/client/src/main/java/com/mifos/feature/client/clientDetails/presentation/ClientDetailsScreen.kt

```diff
@@ -0,0 +1,978 @@
+                    showMenu = false
+                }
+                MifosMenuDropDownItem(option = stringResource(id = R.string.notes)) {
+                    notes(clientId)
+                    showMenu = false
+                }
+                MifosMenuDropDownItem(option = stringResource(id = R.string.pinpoint_location)) {
+                    pinpointLocation(clientId)
+                    showMenu = false
+                }
+                MifosMenuDropDownItem(option = stringResource(id = R.string.survey)) {
+                    survey(clientId)
+                    showMenu = false
+                }
+                MifosMenuDropDownItem(option = stringResource(id = R.string.upload_signature)) {
+                    uploadSignature(clientId)
+                    showMenu = false
+                }
+            }
+        },
+        snackbarHostState = snackbarHostState,
+        bottomBar = {
+            if (client?.active == false) {
+                Button(
+                    onClick = { activateClient(clientId) },
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .heightIn(44.dp)
+                        .padding(start = 16.dp, end = 16.dp),
+                    contentPadding = PaddingValues(),
+                    colors = ButtonDefaults.buttonColors(
+                        containerColor = if (isSystemInDarkTheme()) BluePrimaryDark else BluePrimary
+                    )
+                ) {
+                    Text(text = "Activate Client", fontSize = 16.sp)
+                }
+
+            }
+        }) { padding ->
+        if (showSelectImageDialog) {
+            MifosSelectImageDialog(onDismissRequest = {
+                run {
+                    showSelectImageDialog = !showSelectImageDialog
+                }
+            },
+                takeImage = {
+                    permissionState.permissions.forEach { per ->
+                        when (per.permission) {
+                            Manifest.permission.CAMERA -> {
+                                when {
+                                    per.status.isGranted -> {
+                                        cameraLauncher.launch(cameraImageUri)
+                                    }
+
+                                    else -> {
+                                        permissionState.launchMultiplePermissionRequest()
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                uploadImage = {
+                    galleryLauncher.launch("image/*")
+                }, deleteImage = {
+                    clientDetailsViewModel.deleteClientImage(clientId)
+                    showSelectImageDialog = false
+                })
+        }
+        if (clientNotFoundError) {
+            MifosSweetError(message = "Client Not Found") {
```

Comment:
No hard coded string, and why empty function?

Reply:
I changed